delivery_ups_oca/models/ups_request.py

```diff
@@ -121,7 +121,7 @@ def _quant_package_data_from_picking(self, package, picking, is_package=False):
             },
             "Dimensions": {
                 "UnitOfMeasurement": {"Code": self.package_dimension_code},
-                "Length": str(package.packaging_length),
+                "Length": str(package.length),
```

Comment:
package is a record of the "stock.package.type" (see line 114), I think packaging_length was the correct field (see image)

![image](https://github.com/user-attachments/assets/ef3ccb6e-53f2-41db-b570-d5ab4613c3de)